app/controllers/users_controller.rb

```diff
@@ -104,4 +104,9 @@ def set_user
   def authorize_user
     authorize! :manage, @user
   end
+
+  def update_notice
+    email_changed = @user.email != update_params[:email]
```

Comment:
Did you consider `user.email_changed?` from Rails?

Reply:
Yep, and here is the pitfall - we are not assigning attributes here and thus email attribute is not changed.
And we are updating via #update, not saving, and I totally do not want to make a callback in model on update.

Reply:
What do you think about https://github.com/internetee/auction_center/pull/405/commits/0bc3824309ca92dd4338196d1c80c7fc854fba2b?

Reply:
I made a mistake, updated.

Reply:
I think that's excellent clarification allowing us to use dirty attributes. Thanks a lot!